src/platforms/android/migration.mdx

```diff
@@ -6,6 +6,27 @@ redirect_from:
 description: "Migrating between versions of Sentry's SDK for Android."
 ---
 
+## Migration from `io.sentry:sentry-android-gradle-plugin:3.x.x` to `io.sentry:sentry-android-gradle-plugin:4.0.0`
+
+### Breaking Changes
+
+- Bumped Sentry Android SDK to `7.0.0`. Please, refer to the section below for the detailed changes in the SDK.
+- Renamed `experimentalGuardsquareSupport` flag to `dexguardEnabled`
+
+### Sentry Android SDK Compatibility
+
+Make sure to use Sentry Gradle plugin 4.+ together with the Sentry Android SDK 7.+, otherwise it might crash at runtime due to binary incompatibility. If you can't do that for some reason, you can specify the Sentry version via the plugin config block:
```

Comment:
```suggestion
You must use Sentry Gradle plugin 4.+ together with the Sentry Android SDK 7.+ otherwise, it might crash at runtime due to binary incompatibility. If you can't do that for some reason, you can specify the Sentry version via the plugin config block:
```